utilities/pipelines/dependencies/Microsoft.KeyVault/vaults/parameters/nopr.parameters.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
+      "value": "adp-<<namePrefix>>-az-kv-nopr-001"  // keyvault with purge protection enabled, to be used for encryption
```

Comment:
why not 'nopurge'? too long?

Reply:
with 'nopurge' KV name was too long (27 chars while are 24 allowed)